docker-compose.yml

```diff
@@ -154,3 +163,8 @@ volumes:
     external: true
   sentry-symbolicator:
     external: true
+  sentry-secrets:
```

Comment:
Is this valid yaml? I'd expect you to need to do like, `sentry-smtp: {}`

Reply:
it works so it is valid I guess? :) 🤷‍♂

Reply:
I don't like it.

TIL

```yaml
foo:
bar:
```

is the same as

```json
{
  "foo": null,
  "bar": null
}
```

Reply:
Just saw this sorry, I can change if you think it is confusing?

Reply:
It's fine. I just don't like it because it appears to be invalid syntax. But honestly, most of the yaml syntax you're using looks like invalid syntax. So this is minimal.

Reply:
>But honestly, most of the yaml syntax you're using looks like invalid syntax. 

Lol. If we have a style guide or something, happy to follow that. I think my editor auto formats this using prettier.